dashboard/bower.json

```diff
@@ -14,6 +14,7 @@
     "angular-charts": "0.2.6",
     "angular-cookies": "1.4.8",
     "angular-dropdowns": "1.0.0",
+    "angular-gravatar": "0.2.4",
```

Comment:
what is the associate CQ ?

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14132

Reply:
thx, I've updated the description